app/src/main/java/com/duckduckgo/app/accessibility/AccessibilityActivity.kt

```diff
@@ -38,6 +42,7 @@ import timber.log.Timber
 
 @InjectWith(ActivityScope::class)
 @ContributeToActivityStarter(AccessibilityScreenNoParams::class)
+@ContributeToActivityStarter(AccessibilityScreenParams::class, screenName = "accessibility")
```

Comment:
we can add "accessibility" also to the other `@ContributeToActivityStarter`, so we can have the navigation with empty params also exposed.